config.sh

```diff
@@ -21,14 +24,26 @@ export NEKOPATH=$HVM/versions/neko/$NEKO
 export DYLD_FALLBACK_LIBRARY_PATH=$NEKOPATH
 
 # install haxe if needed
+
 if [ ! -d "$HAXEPATH" ]; then
 	mkdir -p "$HVM/versions/haxe"
 
 	ARCHIVE="$HAXEPATH.tar.gz"
-	URL="http://old.haxe.org/file/haxe-$HAXE-osx.tar.gz"
+
+	case $PLATFORM in
```

Comment:
One place we might need the extra flexibility of a switch statement, is that in 2.* versions of Haxe, the file names were "haxe-2.10-linux.tar.gz", they did not have 32 or 64 bit in the name.  Truth be told I don't even know which they were targeting, and I'm pretty much never going to use 2.x, so I don't mind.

Other than that 2 variables seems easier / cleaner.